server/api/index.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const Recipe = require("./models/recipe");
+
+const { ApolloServer, gql } = require("apollo-server-express");
+
+const typeDefs = gql`
+  input RecipeInput {
+    servings: Int,
+    prepTime: String,
+    cookTime: String,
+    ingredients: [IngredientInput],
+    steps: [String],
+    links: [String],
+    name: String
+  }
+  input IngredientInput {
+    measurement: String!,
+    quantity: Int!,
+    name: String!
+  }
+  type Ingredient {
+    measurement: String!,
+    quantity: Int!,
+    name: String!
+  }
+  type Recipe {
+    servings: Int!,
+    prepTime: String,
+    cookTime: String,
+    ingredients: [Ingredient!],
+    steps: [String!],
+    links: [String],
+    name: String!
+  }
+  type Query { recipes: [Recipe] }
+  type Mutation { createRecipe(input: RecipeInput): Recipe }
+`;
+
+const resolvers = {
+    Query: { 
+      recipes: async() => {
+        return await Recipe.find({});
+      } 
+    },
+    Mutation: {
+      createRecipe: async(parent, args) => {
+        return await Recipe.create(args.input);
+      }
```

Comment:
Unless we require any other validation or logic in this function, this might be simpler
```suggestion
      createRecipe: (parent, args) => Recipe.create(args.input).lean().exec()
```

Reply:
Will this work without the async/await?

Reply:
Yup!

Reply:
Update: It did haha